### PR TITLE
Adds debounce to the counters animation

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds a debounce to the counters animation in order to reduce the browser load.

### CODE REVIEW

File: src/geo/ui/widgets/animate_values.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ module.exports = View.extend({
     var options = opts || {};
     var formatter = options.formatter || d3.format('0,000');
     var templateData = options.templateData || {};
+    var debouceWait = options.debouceWait || 500;
```
Comment (viddo) [inline src/geo/ui/widgets/animate_values.js]: debouce -> debou**n**ce, right?

Comment (javierarce) [inline src/geo/ui/widgets/animate_values.js]: Oops, thanks!

**Comment by javierarce** [comment]
Can you have a look, @xavijam?

**Comment by xavijam** [comment]
debouncing! :+1:

**Comment by javierarce** [comment]
Thankings!

**Pull request merged**